### DISCUSSION TITLE
Exercise 2 results

### DISCUSSION THREAD

**Discussion by ilijafilipovskidm** [sebastian-hofstaetter/teaching#114] [General]
Hi,
Is there an estimated time on when we could expect results for exercise 2? :)
I know it might be might be early, but it would be nice to have an estimate?
Thanks in advance.
Best,
Ilija

**Comment by sophiaalthammer** [comment]
Dear Ilija,

you can expect the results of exercise 2 in around one week.

Best,
Sophia

**Reply by sophiaalthammer** [reply]
The results are now online :)
